governance/roles.md

```diff
@@ -202,41 +209,104 @@ that role for some time informally, then proposed as an on-going role by Pull
 Request which must be approved by a majority of Chairs.  Process clarifications
 are welcome and PRs can be approved by any single Chair.
 
-### Security review facilitator
+### Security Assessment Facilitator
 
 [Security Reviews](/assessments) are part of the ongoing work of the group
-and led by a Security Assessment Facilitator, who will:
-
-* coordinate security review leads for upcoming security assessments.
-* identify and recommend security reviewers.
-* contribute to process improvements.
-* review and merge PRs in the /assessments directory (ensuring co-chair review
-  of significant process changes).
-* triage issues related to security assessments.
-
-### Meeting facilitator
+and led by a security assessment facilitator (referred to in the rest of this
+sub-section as a facilitator). The facilitator is effectively responsible for
+getting new security assessments underway, and providing support on an as-needed
+basis to the security assessment team if anything is impeding the overall review
+process.
+
+The facilitator is responsible for coordinating with the TAG-Security co-chairs
+and deciding on the the order in which security assessments (as submitted in the
+[GitHub issues tracker for
+TAG-Security](https://github.com/cncf/tag-security/issues) will be addressed.
+The facilitator is then responsible for reaching out to the project owners (via
+the GitHub issue said owners previously created in order to request an
+assessment), and coordinating the various requirements as outlined in the
+["Joint Security Review" ticket
+template](https://github.com/cncf/tag-security/issues/new/choose). Once the
+maintainers/owners of the project have been identified, the facilitator is
+responsible for reaching out to the TAG-Security community at large (i.e.
+typically via attending the weekly general meetings or announcing to the
+community [via the `#tag-security`
+channel](https://cloud-native.slack.com/archives/CDJ7MLT8S)), and issue a
+call/request for reviewers (i.e. a single security assessment lead, and at least
+one additional security assessor), and ensuring all reviewers [read the conflict
```

Comment:
`at least one additional security assessor`, do you mean `reviewers` ? Also I would imagine we would need atleast 2 reviewers.

Reply:
Agreed. Updating.